typing/typedtree.ml

```diff
@@ -90,6 +90,11 @@ and expression =
     exp_attributes: attribute list;
    }
 
+and quantified_expression =
+  { qexp_expr: expression;
+    qexp_vars: type_expr list;
```

Comment:
what is the policy for variables with weak polymorphic types? I suppose they are included in the list as well?
it would be good to document the contents of the qexp_vars list, e.g. saying that it only contains rigid type variables, possibly weak.

Reply:
I am talking about the   `'_a` type  variables